pinecone/index_connection_test.go

```diff
@@ -991,6 +991,39 @@ func TestToUsageUnit(t *testing.T) {
 	}
 }
 
+func TestNormalizeHostUnit(t *testing.T) {
+	tests := []struct {
+		name         string
+		host         string
+		expectedHost string
+	}{
+		{
+			name:         "http:// scheme should be removed",
+			host:         "http://this-is-my-host.io",
+			expectedHost: "this-is-my-host.io:443",
+		}, {
+			name:         "https:// scheme should be removed",
+			host:         "https://this-is-my-host.io",
+			expectedHost: "this-is-my-host.io:443",
+		}, {
+			name:         "http:// scheme without a port should be removed",
+			host:         "http://this-is-my-host.io",
+			expectedHost: "this-is-my-host.io:443",
```

Comment:
What is the difference btwn this test and the 1st test? It looks like they both start and end with the same values, no?

Reply:
Ah nothing, accidentally duplicated it!